### ISSUE TITLE
Hassio wont start after upgrade to 2023.10.2 - Silicon Labs Multiprotocol

### ISSUE THREAD

**Issue by carsten-re** [home-assistant/supervisor#4625]
### Describe the issue you are experiencing

System won't start when upgraded to 2023.10.2.
Silicon Labs Multiprotocol is installed, and Supervisor log shows couple of warnings / errors:

```
[31m23-10-13 07:14:52 ERROR (MainThread) [supervisor.misc.tasks] Watchdog found a problem with Home Assistant API![0m
[31m23-10-13 07:14:52 ERROR (MainThread) [supervisor.misc.tasks] Home Assistant watchdog reanimation failed![0m

[33m23-10-13 07:16:49 WARNING (MainThread) [supervisor.addons.options] Option 'otbr_web_enable' does not exist in the schema for Silicon Labs Multiprotocol (core_silabs_multiprotocol)[0m
[33m23-10-13 07:16:49 WARNING (MainThread) [supervisor.addons.options] Option 'otbr_web_enable' does not exist in the schema for Silicon Labs Multiprotocol (core_silabs_multiprotocol)[0m
```

Used version:
Silicon Labs Multiprotocol
Current version: 2.3.2

### What type of installation are you running?

Home Assistant OS

### Which operating system are you running on?

Home Assistant Operating System

### Steps to reproduce the issue

1. Silicon Labs Multiprotocol - Current version: 2.3.2
2. Home Assistant 2023.10.1
3. Do upgrade to 2023.10.2
4. Systemstart will fail, e.g. HACS menu does not appear.

### Anything in the Supervisor logs that might be useful for us?

```txt
23-10-13 07:33:09 INFO (MainThread) [supervisor.api.middleware.security] /backups/6801cb4c/info access from cebe7a76_hassio_google_drive_backup
23-10-13 07:33:09 INFO (MainThread) [supervisor.api.middleware.security] /backups/06d843b7/info access from cebe7a76_hassio_google_drive_backup
23-10-13 07:33:09 INFO (MainThread) [supervisor.api.middleware.security] /backups/e5646763/info access from cebe7a76_hassio_google_drive_backup
23-10-13 07:33:09 INFO (MainThread) [supervisor.api.middleware.security] /backups/826279dd/info access from cebe7a76_hassio_google_drive_backup
23-10-13 07:33:09 INFO (MainThread) [supervisor.api.middleware.security] /backups/33c480be/info access from cebe7a76_hassio_google_drive_backup
23-10-13 07:33:09 INFO (MainThread) [supervisor.api.middleware.security] /backups/f0625681/info access from cebe7a76_hassio_google_drive_backup
23-10-13 07:33:09 INFO (MainThread) [supervisor.api.middleware.security] /backups/25eed65b/info access from cebe7a76_hassio_google_drive_backup
23-10-13 07:33:09 INFO (MainThread) [supervisor.api.middleware.security] /backups/4b7ee51e/info access from cebe7a76_hassio_google_drive_backup
23-10-13 07:33:09 INFO (MainThread) [supervisor.api.middleware.security] /backups/083b7445/info access from cebe7a76_hassio_google_drive_backup
23-10-13 07:33:09 INFO (MainThread) [supervisor.api.middleware.security] /backups/e9392c8b/info access from cebe7a76_hassio_google_drive_backup
23-10-13 07:33:09 INFO (MainThread) [supervisor.api.middleware.security] /backups/d75a87d3/info access from cebe7a76_hassio_google_drive_backup
23-10-13 07:33:09 INFO (MainThread) [supervisor.api.middleware.security] /backups/d646613a/info access from cebe7a76_hassio_google_drive_backup
23-10-13 07:33:09 INFO (MainThread) [supervisor.api.middleware.security] /backups/125b9f43/info access from cebe7a76_hassio_google_drive_backup
23-10-13 07:33:09 INFO (MainThread) [supervisor.api.middleware.security] /backups/626f02d4/info access from cebe7a76_hassio_google_drive_backup
23-10-13 07:33:09 INFO (MainThread) [supervisor.api.middleware.security] /backups/e9a38fd5/info access from cebe7a76_hassio_google_drive_backup
23-10-13 07:33:09 INFO (MainThread) [supervisor.api.middleware.security] /backups/586fec1d/info access from cebe7a76_hassio_google_drive_backup
23-10-13 07:33:13 INFO (MainThread) [supervisor.misc.tasks] All core tasks are scheduled
23-10-13 07:33:13 INFO (MainThread) [supervisor.core] Supervisor is up and running
23-10-13 07:33:13 INFO (MainThread) [supervisor.host.info] Updating local host information
23-10-13 07:33:13 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state running
23-10-13 07:33:13 INFO (MainThread) [supervisor.resolution.checks.base] Run check for ipv4_connection_problem/system
23-10-13 07:33:13 INFO (MainThread) [supervisor.resolution.checks.base] Run check for security/core
23-10-13 07:33:13 INFO (MainThread) [supervisor.resolution.checks.base] Run check for free_space/system
23-10-13 07:33:13 INFO (MainThread) [supervisor.resolution.checks.base] Run check for pwned/addon
23-10-13 07:33:13 WARNING (MainThread) [supervisor.addons.options] Option 'otbr_web_enable' does not exist in the schema for Silicon Labs Multiprotocol (core_silabs_multiprotocol)
23-10-13 07:33:13 INFO (MainThread) [supervisor.resolution.checks.base] Run check for dns_server_failed/dns_server
23-10-13 07:33:13 INFO (MainThread) [supervisor.resolution.checks.base] Run check for no_current_backup/system
23-10-13 07:33:13 INFO (MainThread) [supervisor.resolution.checks.base] Run check for docker_config/system
23-10-13 07:33:13 INFO (MainThread) [supervisor.resolution.checks.base] Run check for dns_server_ipv6_error/dns_server
23-10-13 07:33:13 INFO (MainThread) [supervisor.resolution.checks.base] Run check for multiple_data_disks/system
23-10-13 07:33:13 INFO (MainThread) [supervisor.resolution.checks.base] Run check for trust/supervisor
23-10-13 07:33:13 INFO (MainThread) [supervisor.resolution.check] System checks complete
23-10-13 07:33:13 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state running
23-10-13 07:33:13 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
23-10-13 07:33:13 INFO (MainThread) [supervisor.resolution.fixup] Starting system autofix at state running
23-10-13 07:33:13 INFO (MainThread) [supervisor.resolution.fixups.store_execute_reset] Reset corrupt Store: cebe7a76
23-10-13 07:33:13 INFO (MainThread) [supervisor.store.git] Cloning add-on https://github.com/sabeechen/hassio-google-drive-backup repository
23-10-13 07:33:14 INFO (MainThread) [supervisor.host.services] Updating service information
23-10-13 07:33:14 INFO (MainThread) [supervisor.host.network] Updating local network information
23-10-13 07:33:14 INFO (MainThread) [supervisor.host.sound] Updating PulseAudio information
23-10-13 07:33:14 INFO (MainThread) [supervisor.host.manager] Host information reload completed
23-10-13 07:33:14 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete
23-10-13 07:33:36 WARNING (MainThread) [supervisor.addons.options] Option 'otbr_web_enable' does not exist in the schema for Silicon Labs Multiprotocol (core_silabs_multiprotocol)
23-10-13 07:37:58 WARNING (MainThread) [supervisor.addons.options] Option 'otbr_web_enable' does not exist in the schema for Silicon Labs Multiprotocol (core_silabs_multiprotocol)
23-10-13 07:43:02 WARNING (MainThread) [supervisor.addons.options] Option 'otbr_web_enable' does not exist in the schema for Silicon Labs Multiprotocol (core_silabs_multiprotocol)
23-10-13 07:48:03 WARNING (MainThread) [supervisor.addons.options] Option 'otbr_web_enable' does not exist in the schema for Silicon Labs Multiprotocol (core_silabs_multiprotocol)
23-10-13 07:48:06 WARNING (MainThread) [supervisor.addons.options] Option 'otbr_web_enable' does not exist in the schema for Silicon Labs Multiprotocol (core_silabs_multiprotocol)
23-10-13 07:53:10 WARNING (MainThread) [supervisor.addons.options] Option 'otbr_web_enable' does not exist in the schema for Silicon Labs Multiprotocol (core_silabs_multiprotocol)
23-10-13 07:58:14 WARNING (MainThread) [supervisor.addons.options] Option 'otbr_web_enable' does not exist in the schema for Silicon Labs Multiprotocol (core_silabs_multiprotocol)
23-10-13 07:59:33 WARNING (MainThread) [supervisor.addons.options] Option 'otbr_web_enable' does not exist in the schema for Silicon Labs Multiprotocol (core_silabs_multiprotocol)
23-10-13 08:02:50 INFO (MainThread) [supervisor.backups.manager] Backup 24b7d7ca starting stage addon_repositories
23-10-13 08:02:50 INFO (MainThread) [supervisor.backups.manager] Backup 24b7d7ca starting stage docker_config
23-10-13 08:02:50 INFO (MainThread) [supervisor.backups.manager] Creating new partial backup with slug 24b7d7ca
23-10-13 08:02:50 INFO (MainThread) [supervisor.backups.manager] Backup 24b7d7ca starting stage home_assistant
23-10-13 08:02:50 INFO (MainThread) [supervisor.homeassistant.module] Backing up Home Assistant Core config folder
23-10-13 08:03:00 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
23-10-13 08:03:20 WARNING (MainThread) [supervisor.addons.options] Option 'otbr_web_enable' does not exist in the schema for Silicon Labs Multiprotocol (core_silabs_multiprotocol)
23-10-13 08:03:21 INFO (MainThread) [supervisor.homeassistant.module] Backup Home Assistant Core config folder done
23-10-13 08:03:21 INFO (MainThread) [supervisor.backups.manager] Backup 24b7d7ca starting stage finishing_file
23-10-13 08:03:21 INFO (MainThread) [supervisor.backups.manager] Creating partial backup with slug 24b7d7ca completed
23-10-13 08:03:21 INFO (MainThread) [supervisor.homeassistant.core] Updating Home Assistant to version 2023.10.2
23-10-13 08:03:21 INFO (MainThread) [supervisor.docker.interface] Updating image ghcr.io/home-assistant/qemux86-64-homeassistant:2023.10.1 to ghcr.io/home-assistant/qemux86-64-homeassistant:2023.10.2
23-10-13 08:03:21 INFO (MainThread) [supervisor.docker.interface] Downloading docker image ghcr.io/home-assistant/qemux86-64-homeassistant with tag 2023.10.2.
23-10-13 08:03:32 INFO (MainThread) [supervisor.api.middleware.security] /supervisor/info access from cebe7a76_hassio_google_drive_backup
23-10-13 08:03:32 INFO (MainThread) [supervisor.api.middleware.security] /backups access from cebe7a76_hassio_google_drive_backup
23-10-13 08:03:32 INFO (MainThread) [supervisor.api.middleware.security] /backups/24b7d7ca/info access from cebe7a76_hassio_google_drive_backup
23-10-13 08:03:32 INFO (MainThread) [supervisor.api.middleware.security] /supervisor/info access from cebe7a76_hassio_google_drive_backup
23-10-13 08:03:32 INFO (MainThread) [supervisor.api.middleware.security] /backups access from cebe7a76_hassio_google_drive_backup
23-10-13 08:03:33 INFO (MainThread) [supervisor.api.middleware.security] /backups/826279dd access from cebe7a76_hassio_google_drive_backup
23-10-13 08:03:33 INFO (MainThread) [supervisor.backups.manager] Removed backup file 826279dd
23-10-13 08:03:33 INFO (MainThread) [supervisor.api.middleware.security] /backups/24b7d7ca/download access from cebe7a76_hassio_google_drive_backup
23-10-13 08:03:33 INFO (MainThread) [supervisor.api.backups] Downloading backup 24b7d7ca
23-10-13 08:04:00 INFO (MainThread) [supervisor.api.middleware.security] /supervisor/info access from cebe7a76_hassio_google_drive_backup
23-10-13 08:04:00 INFO (MainThread) [supervisor.api.middleware.security] /backups access from cebe7a76_hassio_google_drive_backup
23-10-13 08:04:00 INFO (MainThread) [supervisor.api.middleware.security] /supervisor/info access from cebe7a76_hassio_google_drive_backup
23-10-13 08:04:00 INFO (MainThread) [supervisor.api.middleware.security] /backups access from cebe7a76_hassio_google_drive_backup
23-10-13 08:04:44 INFO (SyncWorker_4) [supervisor.docker.manager] Stopping homeassistant application
23-10-13 08:05:02 INFO (SyncWorker_4) [supervisor.docker.manager] Cleaning homeassistant application
23-10-13 08:05:02 INFO (MainThread) [supervisor.homeassistant.module] Update pulse/client.config: /data/tmp/homeassistant_pulse
23-10-13 08:05:02 INFO (MainThread) [supervisor.docker.homeassistant] Starting Home Assistant ghcr.io/home-assistant/qemux86-64-homeassistant with version 2023.10.2
23-10-13 08:05:02 INFO (MainThread) [supervisor.homeassistant.core] Wait until Home Assistant is ready
23-10-13 08:05:03 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state running
23-10-13 08:05:03 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
23-10-13 08:05:04 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/esphome/home-assistant-addon repository
23-10-13 08:05:04 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/zigbee2mqtt/hassio-zigbee2mqtt repository
23-10-13 08:05:04 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/hassio-addons/repository repository
23-10-13 08:05:04 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/albertogeniola/ha-meross-local-broker repository
23-10-13 08:05:04 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/home-assistant/addons repository
23-10-13 08:05:04 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/sabeechen/hassio-google-drive-backup repository
23-10-13 08:05:04 INFO (MainThread) [supervisor.updater] Fetching update data from https://version.home-assistant.io/stable.json
23-10-13 08:05:05 INFO (MainThread) [supervisor.store] Loading add-ons from store: 77 all - 0 new - 0 remove
23-10-13 08:05:05 INFO (MainThread) [supervisor.store] Loading add-ons from store: 77 all - 0 new - 0 remove
23-10-13 08:05:09 WARNING (MainThread) [supervisor.addons.options] Option 'otbr_web_enable' does not exist in the schema for Silicon Labs Multiprotocol (core_silabs_multiprotocol)
23-10-13 08:05:13 WARNING (MainThread) [supervisor.misc.tasks] Watchdog miss API response from Home Assistant
```

### System Health information

This will not load entierly.

![grafik](https://github.com/home-assistant/supervisor/assets/59530556/fbc98371-511d-436d-ada4-eac1ad14f51d)

### Supervisor diagnostics

is not available

**Comment by carsten-re** [comment]
same behavior with 2023.10.3

**Comment by carsten-re** [comment]
Problem is now solved with version 2023.10.5
